int64 status
string info
